make/generate-markdown

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -o errexit
+
+GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+# Force an error if $HOME isn't set
+home=${HOME:?"HOME isn't set"}
+
+set -o nounset
+
+cd ${GIT_ROOT}
+./fissile docs markdown
+# Change expanded home directory back to '~'
+for x in $(git status | awk 'modified:\s+docs/ {print $2}') ; do
+    sed -i 's@'${HOME}'@~@g' $x
+done
+# Then check for date changes only, and revert if that's all we have.
+for x in $(git status | awk 'modified:\s+docs/ {print $2}') ; do
+    numUnmatched=$(git diff $x | grep -v -E '^[-+]###### Auto generated by spf13/cobra on [0-9]+-[a-zA-Z]+-[0-9]{4}' | wc -l)
+    case $numUnmatched in
+	0) git checkout -- $x ;;
+	*) #keep diffs in $x ;;
```

Comment:
Fixed in d7dc0ed (by putting the ';;' on a separate line, so the code is slightly more maintainable.